src/index.js

```diff
@@ -80,7 +81,7 @@ export class ESLintWebpackPlugin {
 
       // @ts-ignore
       const processModule = (module) => {
-        const file = module.resource;
+        const file = parse(module.resource).pathname;
```

Comment:
Just use `indexOf('?')`, it is faster and still safe

Reply:
Assuming you mean `.split('?')[0]`, I speak to that in the PR body but it seems like `url.parse` is equally prone to the same edge-cases (eg. `/file-name-with-?-this-is-ignored.js`).

Will update.